core/controllers/translator.py

```diff
@@ -179,6 +179,7 @@ def post(self, exploration_id):
 
 
 class ExplorationTranslationHandler(base.BaseHandler):
+    """Handles given exploration."""
```

Comment:
I don't understand what this means...it's not specific enough. What does this handler actually do, conceptually?

Reply:
Done.